solr/server/resources/log4j2.xml

```diff
@@ -39,9 +39,9 @@
       </PatternLayout>
       <Policies>
         <OnStartupTriggeringPolicy />
-        <SizeBasedTriggeringPolicy size="32 MB"/>
+        <SizeBasedTriggeringPolicy size="${sys:solr.log.rolloversize:-32 MB}"/>
       </Policies>
-      <DefaultRolloverStrategy max="10"/>
+      <DefaultRolloverStrategy max="${sys:solr.log.filestokeep:-10}"/>
```

Comment:
While working with rollover configuration for requestlogs, I also thought I'd make the size and number of files for the ordinary solr.log configurable. New sys.props: `-Dsolr.log.rolloversize` and `-Dsolr.log.filestokeep`. They still default to `32Mb` and `10` respectively. This also needs testing and documenting in refguide.

Reply:
I think I'll separate out this (unrelated) change into its own PR for better clarity.

Reply:
Reverted. See instead #537